### ISSUE TITLE
Not able to install Elasticsearch 5.5.2 in windows 2016 server

### ISSUE THREAD

**Issue by pratheeprpa** [elastic/windows-installers#335]
I had installed Elastic search in windows server 2016 server and along with kibana. Kibana url was not accessing in browser. I have uninstalled elastic search and try to reinstalling but i am getting error at end of the installation of ES 5.5.2. can anyone help on this

**Comment by ygel** [comment]
Hello @pratheeprpa, thank you for reporting this. First, is it possible for you to use latest installer version from https://www.elastic.co/downloads/elasticsearch, currently 7.4.0? If it's not, we'll need MSI logs to diagnose the issue.

To generate logs run:
 ```msiexec /l!*vx "<full path to log file to be created>" /i "<full path to msi file>"```

**Comment by AlexanderAnatolich** [comment]
Hello. I have this problem too.
Elastic ver. 7.5.0, Nest ver. 7.5.1
I encountered the problem of deploying "elasticsearch" on an Azure virtual machine running Windows Server 2016 Datacenter.
My API is written in .Net Core 3.1. When I try to connect to the "elasticsearch" client https://prnt.sc/qdflqr, I get the error https://prnt.sc/qdfjr4.
With this, the same configurations work successfully under Windows 10 both on the local machine and on the VirtualBox virtual machine (Windows 10).
I try to solve this problem more two days. Disabling the firewall does not solve this problem.
During installation, I did not set any passwords.
If I do this, then I get an installation error. It can be see from the logs that during the installation, attempts to connect to the "elasticsearch" client are unsuccessful.

[Log.txt](https://github.com/elastic/windows-installers/files/3987476/Log.txt)
